src/click.js

```diff
@@ -65,10 +65,11 @@ function clickElement(element, init, {clickCount}) {
       continueDefaultHandling &&
       element !== element.ownerDocument.activeElement
     ) {
-      if (previousElement && !isFocusable(element)) {
+      const closestFocusable = findClosest(element, isFocusable)
+      if (previousElement && !closestFocusable) {
```

Comment:
What will happen when `previousElement` and `closestFocusable` is pointing to the same element?

Reply:
`focus` is safe to call
https://github.com/testing-library/user-event/blob/7a5c51e7f89c4ab0592174eff591b9e9275ff81c/src/focus.js#L6-L7

Reply:
I removed the check above. It is redundant and gives the impression we'd have to guard the call to `focus`.

Reply:
With this implementation it makes sense, thank you for the explanation!